TypeError: PyDataFrame.__arrow_c_stream__() missing 1 required positional argument: 'requested_schema' with polars 1.32.3

I started getting this error after a recent Polars update: 

```
        ).run(
>           pt.DataFrame(parents_df).set_model(DummyInItem),
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
            ParentStatefulTransform,
        )

tests/test_ray_utils.py:167: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.11/site-packages/patito/polars.py:446: in set_model
    return model.DataFrame(self._df)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.11/site-packages/polars/dataframe/frame.py:450: in __init__
    self._df = pycapsule_to_frame(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <builtins.PyDataFrame object at 0x7efd880f5570>

    def pycapsule_to_frame(
        obj: Any,
        *,
        schema: SchemaDefinition | None = None,
        schema_overrides: SchemaDict | None = None,
        rechunk: bool = False,
    ) -> DataFrame:
        """Convert PyCapsule object to DataFrame."""
        if hasattr(obj, "__arrow_c_array__"):
            # This uses the fact that PySeries.from_arrow_c_array will create a
            # struct-typed Series. Then we unpack that to a DataFrame.
            tmp_col_name = ""
            s = wrap_s(PySeries.from_arrow_c_array(obj))
            df = s.to_frame(tmp_col_name).unnest(tmp_col_name)
    
        elif hasattr(obj, "__arrow_c_stream__"):
            # This uses the fact that PySeries.from_arrow_c_stream will create a
            # struct-typed Series. Then we unpack that to a DataFrame.
            tmp_col_name = ""
>           s = wrap_s(PySeries.from_arrow_c_stream(obj))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E           TypeError: PyDataFrame.__arrow_c_stream__() missing 1 required positional argument: 'requested_schema'

/usr/local/lib/python3.11/site-packages/polars/_utils/pycapsule.py:41: TypeError
```

Please let me know if I should report it to `polars` instead.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,7 +15,7 @@ keywords = [
 ]
 dependencies = [
     "pydantic>=2.7.0",
-    "polars>=1.10.0",
+    "polars>=1.32.0",
     "typing-extensions",
 ]
 

--- a/src/patito/polars.py
+++ b/src/patito/polars.py
@@ -443,7 +443,7 @@ class DataFrame(pl.DataFrame, Generic[ModelType]):
             >>> casted_classes.validate()
 
         """
-        return model.DataFrame(self._df)
+        return model.DataFrame._from_pydf(self._df)
 
     def unalias(self: DF) -> DF:
         """Un-aliases column names using information from pydantic validation_alias.
@@ -537,9 +537,8 @@ class DataFrame(pl.DataFrame, Generic[ModelType]):
 
         """
         if columns is not None:
-            # I get a single null row if I try to use super() here, so go via
-            # pl.DataFrame instead.
-            return self._from_pydf(pl.DataFrame(self._df).drop(columns)._df)
+            # Use super() to call polars DataFrame drop method directly
+            return self._from_pydf(super().drop(columns)._df)
         else:
             return self.drop(list(set(self.columns) - set(self.model.columns)))
 

